lib/manageiq/appliance_console/message_configuration_server.rb

```diff
@@ -170,17 +182,16 @@ def create_server_properties
       end
 
       def deactivate
-        configure_messaging_type("miq_queue") # Settings.prototype.messaging_type = 'miq_queue'
-        restart_evmserverd
-        remove_installed_files
+        super
+
         unconfigure_firewall
         deactivate_services
       end
 
       def unconfigure_firewall
         say(__method__.to_s.tr("_", " ").titleize)
 
-        AwesomeSpawn.run!("firewall-cmd --remove-port=9093/tcp --permanent") # secure
+        AwesomeSpawn.run!("firewall-cmd --remove-port=#{server_port}/tcp --permanent") # secure
```

Comment:
This feels like a potential command injection issue.  We shouldn't be passing params this way with AwesomeSpawn (yes, I see it was that way before), but it's an actual issue after this change in that there is potential user injection.

```suggestion
        AwesomeSpawn.run!("firewall-cmd", :params => {:remove_port => "#{server_port}/tcp", :permanent => nil}) # secure
```